cmd/config.go

```diff
@@ -24,6 +24,16 @@ import (
 	"github.com/minio/mc/pkg/console"
 )
```

Comment:
I see this file contains configuration directory which contains configuration files and  subdirectories.  May be configdir.go?